app/models/site_upload.rb

```diff
@@ -17,7 +17,7 @@
 class SiteUpload < ApplicationRecord
   has_attached_file :file
 
-  validates_attachment_content_type :file, content_type: /\Aimage\/.*\z/
+  validates_attachment_content_type :file, content_type: /\Aimage\/.*\z/, presence: true
```

Comment:
I don't think this will work. It needs to be an extra `validates :file, presence: true`

Reply:
Alright, should be fixed now.